test/mailers/previews/last_seen_mailer_preview.rb

```diff
@@ -1,5 +1,18 @@
-class LastSyncedMailerPreview < ActionMailer::Preview
-    def last_seen_more_than_3_days
-        
+class LastSeenMailerPreview < ActionMailer::Preview
+    require 'mailer_service/last_seen_last_sync_service'
+
+    def summary_of_last_seen
+        result = LastSeenLastSyncService.processed_data
+      
+        if result[:last_seen].present? && result[:last_seen][:districts].any?
+            LastSeenMailer.summary_of_last_seen(result[:last_seen]).deliver_now
```

Comment:
Remove the `.deliver_now` for it so that an email and valid smtp details are not required with doing the preview.
```suggestion
            LastSeenMailer.summary_of_last_seen(result[:last_seen])
```